pkg/pipelines/webhook/webhook_test.go

```diff
@@ -160,6 +165,6 @@ func TestGetAccessToken(t *testing.T) {
 		if token != tt.expectedToken {
 			t.Fatalf("getAcessToken returned %v, expected %v", token, tt.expectedToken)
```

Comment:
This is within a table test, you want to use `Errorf` in table tests, because you want to see all tests that fail, same above.